web-landing/index.provisioned.html

```diff
@@ -8,24 +8,35 @@
   </head>
   <!-- Download the app! -->
   <body>
-  
     <img src="moto-sync-logo.png" class="moto-sync-logo">
-  
+
     <div class="card download-links-card">
       <h2 class="header">Please download the <span class="bold">moto</span>sync app to manage your network and access free live support</h2>
-      <div class="download-links">
-        <a href="http://motorolanetwork.com/motosyc">Download <span class="bold">moto</span>sync</a>
-      </div>
-      <div class="download-links" style="margin-top: 16px;">
+      <a href="http://motorolanetwork.com/pages/motosync" class="primary-button m-auto">
+        <span>Download <span class="bold">moto</span>sync</span>
+      </a>
+      <div class="qr-section" style="margin-top: 32px;">
         <h2 class="paragraph">Or scan with your mobile device</h2>
-        <img src="qr-motorolanetwork.com-motosync.png" class="qr-code">
+        <img src="qr-code.png" class="qr-code">
       </div>
     </div>
+
+    <!-- Insert link here -->
```

Comment:
Please put the modem management html inside a comment for now.  This file is used for both routers and cablemodem gateways  - my plan is to replace the <!-- Placeholder --> with the cable modem link when the file is packaged for a cablemodem gateway.  I'll do that step now that you have the html done for me.

Reply:
@Tom-Keddie Gotcha. Updated the PR